src/Attestation/documentation/upcoming-breaking-changes.md

```diff
@@ -0,0 +1,176 @@
+<!--
+    Please leave this section at the top of the breaking change documentation.
+
+    New breaking changes should go under the section titled "Upcoming Breaking Changes", and should adhere to the following format:
+
+    # Upcoming Breaking Changes
+
+    ## Release X.0.0 - January 2017
+
+    The following cmdlets were affected this release:
+
+    **Cmdlet 1**
+    - Description of what has changed
+
+    ```powershell
+    # Old
+    # Sample of how the cmdlet was previously called
+
+    # New
+    # Sample of how the cmdlet should now be called
+    ```
+
+    Note: the above section follows the template found in the link below: 
+
+    https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes/breaking-change-template.md
+-->
+
+# Upcoming Breaking Changes
+The following cmdlets were affected this release:
```

Comment:
@shleiAmy you can remove all of the below content as it doesn't apply to your service

Reply:
removed